Firebase.Invites/source/Firebase.Invites/ApiDefinition.cs

```diff
@@ -29,9 +29,9 @@ interface IInviteDelegate
 	}
 
 	// @protocol FIRInviteDelegate <NSObject>
-	[Model]
+	[Model (AutoGeneratedName = true)]
 	[Protocol]
-	[BaseType (typeof (NSObject), Name = "FIRInviteDelegate")]
```

Comment:
You removed the native name `FIRInviteDelegate` so the protocol won't be found at all